apps/docs/src/content/docs/libraries/env.mdx

```diff
@@ -0,0 +1,152 @@
+---
+title: env
+description: Type-safe environment variable provider.
```

Comment:
Nice. Optionally, Fumadocs has a component for this you can use.

![image](https://github.com/user-attachments/assets/14b82044-7628-4da7-8a56-20ef536f600b)